app/Http/Controllers/FashionController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Fashion;
+use App\Traits\Common;
+
+class FashionController extends Controller
+{
+    use Common;
+    /**
+     * Display a listing of the resource.
+     */
+
+    public function index(){
+        /* $products = Fashion::orderBy('created_at', 'desc')
+        ->take(3)
+        ->get(); */
+        //$products =Fashion::orderBy('id', 'desc')->take(3)->get();
+        $products =Fashion::latest()->take(3)->get();
+        return view('index', compact('products'));
+    }
+
+    public function about(){
```

Comment:
personally, i don't like to put the public pages in the crud controller and i will tell you why

if you open the about page, you will find a section called meet our team, is this section belongs to products?

The way i prefer to work with is that and a send it as text file in the group called organizeFiles

if your website has two sections
Public pages ==> like the template of Fashion
Admin pages ==> like the crud of cars, products ...etc

for controllers
- i separate my controllers in directory called `Admin` for crud controllers or any pages for admin only
- for the public pages, it depends on the pages if there are a lot of pages i make more than one controller each one serve part of public pages & if the number of pages is small, i make one controller e.g. `PublicController` and make function for each page 
      - for example
 ```php
class PublicController {
    public function index() {
        // for index page
    }
    public function about() {
        // for about page
     }
    ...
}
```